nix/sources.json

```diff
@@ -0,0 +1,39 @@
+{
+    "cardano-node": {
```

Comment:
Does this need to be a dependency? There's currently no integration test, and I can't see it being used